schema/unsupported_features.json

```diff
@@ -2,15 +2,15 @@
     "setup_types": [
         {
             "name":"virtual",
-            "unsupported_features": ["alerts","health", "cluster_management", "lyve_pilot"]
+            "unsupported_features": ["lyve_pilot"]
         },
         {
             "name":"5u84",
-            "unsupported_features": []
+            "unsupported_features": ["lyve_pilot"]
```

Comment:
why "lyve_pilot" should be disabled in 5u84 and PODS by default?

Reply:
This change has been done after discussion with @kupranay (Pranay). Initially it was not there.

Reply:
As discussed default is lyve_pylot disabled. However on build time we are taking a decision to update config file